mock/ecs_pod.go

```diff
@@ -2,30 +2,51 @@ package mock
 
 import (
 	"context"
-	"errors"
 
 	"github.com/evergreen-ci/cocoa"
 )
 
 // ECSPod provides a mock implementation of a cocoa.ECSPod. By default, it is
-// backed by a mock ECS client.
-type ECSPod struct{}
+// backed by a mock ECS pod.
+type ECSPod struct {
+	cocoa.ECSPod
+
+	Input      context.Context
```

Comment:
It's not necessary to record the input context in this case since it's not really a parameter or option. The context is primarily useful to stop a running operation if it's taking too long.

Reply:
Alright, I'll take it out.